pages/made-in-youngstown.tsx

```diff
@@ -1,13 +1,21 @@
 import Layout from "../components/Layout";
 import Companies from "../components/Companies";
 
-// add filters for type of work done, consulting or not
+const pageDescription =
+  "Local tech companies in and around the Youngstown OH area.";
 
 export default function MadeInYoungstown() {
   return (
-    <Layout page="Made In Youngstown">
+    <Layout page="Made In Youngstown" description={pageDescription}>
       <h1 className="text-3xl p-2">Made In Youngstown</h1>
-      <h2 className="text-2xl p-2">Local Tech Companies</h2>
+      <h2 className="sm:text-2xl text-xl p-2">
```

Comment:
Let's just have one page description here

A directory of Youngstown-area organizations that employ technology professionals, sourced from the Code Youngstown community. Includes local consultancies that can help you with your next app, website, or anything else.